client/packages/invoices/src/OutboundShipment/DetailView/columns.ts

```diff
@@ -26,6 +27,18 @@ const notePopoverColumn = getNotePopoverColumn<
   OutboundLineFragment | OutboundItem
 >();
 
+const isDefaultPlaceholderRow = (row: OutboundLineFragment) =>
+  row.type === InvoiceLineNodeType.UnallocatedStock && !row.numberOfPacks;
+
+const getPackSize = (row: OutboundLineFragment) =>
+  isDefaultPlaceholderRow(row) ? '' : row.packSize;
+
+const getNumberOfPacks = (row: OutboundLineFragment) =>
+  isDefaultPlaceholderRow(row) ? '' : row.numberOfPacks;
+
+const getUnitQuantity = (row: OutboundLineFragment) =>
```

Comment:
possibly overboard, doing it for all the columns.. but it seems odd having `$0.00` show for the placeholders, and `0` for quantity etc.